templates/game.eex

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Elixir TicTacToe</title>
+    <style>
+
+body {
+  background-color: black;
+  color: white;
+}
+
+h1 {
+  text-align: center;
+}
+
+#board {
+  background-color: #4d4d4d;
+  width: 400px;
+  height: 400px;
+  margin: auto;
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+}
+
+.row {
+  margin: auto;
+  width: 99%;
+  height: 32%;
+  display: flex;
+  justify-content: space-around;
+}
+
+.cell {
+  width: 30%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #808080;
+  color: black;
+}
+
+.cell-mark {
+  text-align: center;
+  font-size: 50px;
+}
+
+form, button {
+    width: 100%;
+    height: 100%;
+    background-color: #808080;
+    border: none;
+    cursor: crosshair;
+    overflow: hidden;
+    outline:none;
+}
+    </style>
+  </head>
+
+  <body>
+
+    <h1>It is <%= current_player.mark %>'s turn, please pick a move</h1>
+
+    <div id="board">
+      <%= for row <- rows do %>
+        <div class="row">
+          <%= for cell <- row do %>
+            <div class="cell">
+              <% {cell_mark, cell_index} = cell %>
```

Comment:
There's a little bit of logic in this view that could be pushed to a presenter.

If you change the `:empty` atom to something else, then it may not be totally obvious that this file also should be changed.

Could you create a cell presenter, and then you could have properties on it like `cell.empty?`, `cell.index`, `cell.mark`.